lib/modules/model/types/ModelApi.ts

```diff
@@ -71,6 +73,24 @@ export interface ApiModelWriteMeasureRequest extends ModelsControllerRequest {
 }
 export type ApiModelWriteMeasureResult = KDocument<MeasureModelContent>;
 
+export interface ApiModelUpdateAssetRequest extends ModelsControllerRequest {
+  action: "updateAsset";
+
+  _id: string;
+
+  body: {
+    engineGroup: string;
+    model: string;
```

Comment:
Maybe put `engineGroup` and `model` under `_id` as they should not be updated, right now it gives the feeling that we are able to update these values